algoliasearch-apache-uber/pom.xml

```diff
@@ -17,6 +17,27 @@
     </dependencies>
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
```

Comment:
Could you please add a `<version>2.4</version>` just after the `<artifactId>` declaration. Otherwise, a warning is generated at each Maven run.

I know there are more recent versions of `maven-jar-plugin` but since `2.4` is the one we use in the top `pom.xml` file, I suggest to keep the same version for this PR and open a new PR whenever we will want to upgrade it.